File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/lectures/LectureSearchSuggestionProvider.java
package de.tum.in.tumcampusapp.component.tumui.lectures;

import android.content.SearchRecentSuggestionsProvider;

public class LectureSearchSuggestionProvider extends SearchRecentSuggestionsProvider {
    public final static String AUTHORITY = "de.tum.in.tumcampusapp.component.tumui.lectures.LectureSearchSuggestionProvider";

    public LectureSearchSuggestionProvider() {
        setupSuggestions(AUTHORITY, 1);
    }
}

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/lectures/LectureSearchSuggestionProvider.kt
package de.tum.`in`.tumcampusapp.component.tumui.lectures

import android.content.SearchRecentSuggestionsProvider

import de.tum.`in`.tumcampusapp.component.tumui.lectures.activity.LecturesPersonalActivity

/**
 * Suggestion provider for [LecturesPersonalActivity]
 */
class LectureSearchSuggestionProvider : SearchRecentSuggestionsProvider() {
    init {
        setupSuggestions(AUTHORITY, 1)
    }

    companion object {
        const val AUTHORITY = "de.tum.in.tumcampusapp.component.tumui.lectures.LectureSearchSuggestionProvider"
    }
}